pkg/controller/directvolumemigration/rsync.go

```diff
@@ -553,6 +576,43 @@ func (t *Task) createRsyncTransferPods() error {
 	return nil
 }
 
+func getPodConfig(client k8sclient.Client, cpu_limit string, memory_limit string, cpu_request string, memory_request string, ) (corev1.ResourceList, corev1.ResourceList, error) {
+	podConfigMap := &corev1.ConfigMap{}
+	err := client.Get(context.TODO(), types.NamespacedName{Name: "migration-controller", Namespace: migapi.OpenshiftMigrationNamespace}, podConfigMap)
+	if err != nil {
+		return nil, nil, err
+	}
+	limits := corev1.ResourceList{
+		corev1.ResourceMemory: resource.MustParse("1Gi"),
+		corev1.ResourceCPU:    resource.MustParse("1"),
+	}
+	if _, exists := podConfigMap.Data[cpu_limit]; exists {
+		cpu := resource.MustParse(podConfigMap.Data[cpu_limit])
+		if _, exists := podConfigMap.Data[memory_limit]; exists {
```

Comment:
why is this nested if? should the if here and if 2 lines above be independent?

Reply:
The idea was to check for both the values and then set, but now looking at it again I think we can remove nested if.